workspaces/ui/src/components/loaders/ApiLoader.js

```diff
@@ -39,7 +39,7 @@ export function ApiSpecServiceLoader(props) {
     }
   }, [service]);
 
-  if (!service) {
+  if (!events) {
```

Comment:
If this is a separate fix, we should merge it separately

Reply:
makes sense — made #293 to address this